src/core/server/graph/schema/schema.graphql

```diff
@@ -3661,6 +3668,20 @@ type Comment @cacheControl(maxAge: 5) {
   """
   parentCount: Int!
 
+  """
+  allChildComments is a CommentsConnection that includes all comments that are
+  children of a comment (i.e. that have the comment included in their ancestorIDs)
+  """
+  allChildComments(
+    first: Int = 10
+    orderBy: COMMENT_SORT = CREATED_AT_DESC
+  ): CommentsConnection!
```

Comment:
Do we think it's clearer readability wise for this to be as it is named:`allChildComments` versus a flag/option called `all` (or similar) on the existing `comments` resolver?

Reply:
Nevermind, I thought this was on the Query type and not the Comment type. This makes more sense.